packages/typespec-azure-resource-manager/README.md

```diff
@@ -94,7 +94,7 @@ This decorator is used either on a namespace or a version enum value to indicate
 the version of the Azure Resource Manager common-types to use for refs in emitted Swagger files.
 
 ```typespec
-@Azure.ResourceManager.armCommonTypesVersion(version: valueof string | EnumMember)
+@Azure.ResourceManager.armCommonTypesVersion(version: EnumMember)
```

Comment:
isn't that a breaking change?

Reply:
Yes. There were handful of using "v5" string. But let me add it back and do a look up.